services/merak-topo/database/database.go

```diff
@@ -50,11 +52,13 @@ func ConnectDatabase() error {
 func SetValue(key string, val interface{}) error {
 	j, err := json.Marshal(val)
```

Comment:
would be great to add metrics here, so we can see the call count and latency of this function, same to other apis

Reply:
but only metrics, not logs.

Reply:
```
	var err error
	defer metrics.GetMetrics(m, &err)()
```
https://github.com/futurewei-cloud/merak/blob/2d37c70af68c86a3bed8332b70b803ba956f8def/services/merak-agent/evm/alcor.go#L497

Reply:
not use it this time, will try it in future, tks!